pages/index.js

```diff
@@ -16,67 +16,123 @@ export default function Home({
     sport: {
       basketball: false,
       fitness: false,
-      other: false,
-      soccer: false,
+      boule: false,
+      football: false,
       tennis: false,
-      volleyball: false,
+      beachvolleyball: false,
+      skateboard: false,
+      tischtennis: false,
     },
-    city: { Münster: false, Düsseldorf: false },
-    rating: { 1: false, 2: false, 3: false, 4: false, 5: false },
   });
-
-  //show / hide Filter Menu
+  const [filterDataOther, setFilterDataOther] = useState({
+    lighting: false,
+    wheelchair: false,
+    outdoor: false,
+    public: false,
+  });
+  const [cityFilter, setCityFilter] = useState();
+  const [sortData, setSortData] = useState();
+  const [params, setParams] = useState(
+    new URL("http://localhost:3000/api/locations/?")
+  );
   const [isShowingFilterMenu, setIsShowingFilterMenu] = useState(false);
 
+  //show / hide Filter Menu
```

Comment:
the name of the function is speaking by itself -> you don't need the comment here